src/DataStructures/Tensor/Expressions/SpatialSpacetimeIndex.hpp

```diff
@@ -133,5 +134,43 @@ template <typename TensorIndexTypeList, typename SpatialSpacetimeIndexPositions>
 using replace_spatial_spacetime_indices = tmpl::fold<
     SpatialSpacetimeIndexPositions, TensorIndexTypeList,
     replace_spatial_spacetime_indices_helper<tmpl::_state, tmpl::_element>>;
+
+/// \brief Given a number of tensor indices of two tensors and the positions of
+/// each tensor's spacetime indices for which a generic spatial index was used,
+/// compute the shift in the multi-index values from the first tensor's
+/// multi-indices to the second's
+///
+/// \details
+/// Example: If we have \f$R_{ijk} + S{ijk}\f$, where  \f$R\f$'s first and
+/// 2nd indices are spacetime and \f$S\f$' first index and third index are
+/// spacetime, let \f$i = 3\f$, \f$j = 5\f$, and \f$k = 7\f$. The multi-index
```

Comment:
Shouldn't all these numbers be less than 4? Or maybe bigger than 0 and less than 4?

Reply:
That'a a good point. I'll update the example